Init-ALEnvironment.ps1

```diff
@@ -88,8 +90,13 @@ function Init-ALEnvironment
                 $credentials = Get-Credential -Message "Enter password you want to use" -UserName $Username
             }
         }
+
         $myscripts = @(@{'MainLoop.ps1' = 'while ($true) { start-sleep -seconds 10 }'})
 
+        if($customScripts) {

```

Comment:
Why it is there when in next change the $myscripts is overwritten with $customScripts at all?